maven-plugin/src/main/java/org/openmrs/maven/plugins/CreateProject.java

```diff
@@ -269,6 +281,10 @@ private void createModule() throws MojoExecutionException {
 					"What is the lowest version of the Reference Application (-D%s) you want to support?", refapp, "refapp",
 					"2.4");
 			archetypeArtifactId = SDKConstants.REFAPP_ARCH_ARTIFACT_ID;
+		} else if (TYPE_CONTENT_PACKAGE.equals(type)) {
+			contentpackage = wizard.promptForValueIfMissingWithDefault("What is the lowest version of the content package (-D%s) you want to support?", contentpackage, "contentpackage",
```

Comment:
I don't really understand what this question is prompting for?

Reply:
It didn't make sense to me either.  Was just going by the convention overthere,  I can skip the prompt.

Thanks

Reply:
Oh, yeah. For other things we care about the platform version, etc. We currently don't for content packages, but maybe we should?